### DISCUSSION TITLE
Automatically redirect to SSO for users from a domain

### DISCUSSION THREAD

**Discussion by nathanclayton** [goauthentik/authentik#3184] [Q&A] [answered]
Is there a way to automatically redirect all users from a given domain (e.g. user@foo.com) to a specific SSO provider and users from a different domain (user@bar.com) to a different SSO provider?

I'd like to set it up so that all internal users are automatically redirected to the main corporate SSO solution, and clients would be redirected to their respective SSO solution.

Seeing as how we handle multiple clients, all with their own SSO, I'd like to be able to do this via the user's email so that there isn't a wall of SSO icons for the end user to pick from.

**Comment by BeryJu** [comment, marked as answer]
See https://goauthentik.io/docs/flow/examples/snippets

**Reply by nathanclayton** [reply]
Thank you for the fast reply - I'll give this a try shortly.
